### ISSUE TITLE
Add data-no-csrf to our form

### ISSUE THREAD

**Issue by wresuolc** [mozilla/send#631]
Let's add `data-no-csrf` to the form on https://send.firefox.com/.  It would become:

```
      <form method="post" action="upload" enctype="multipart/form-data" data-no-csrf>
```

**Comment by wresuolc** [comment]
Technically, it doesn't do anything anyway. :)  It's just a flag that says "we know there isn't a csrf token here and we're doing it on purpose".  It's a mozilla policy, not a web thing.

**Comment by dannycoates** [comment]
fixed in 2e233da16d1b62b569e99802b494a3765ef04d64
